include/circt/Dialect/MSFT/DeviceDB.h

```diff
@@ -77,6 +77,14 @@ class PlacementDB {
     Operation *op;
   };
 
+  /// Contains the order to iterate in each dimension for walkPlacements.
+  enum Direction { NONE = 0, ASC = 1, DESC = 2 };
+  struct WalkOrder {
```

Comment:
Please add a note that the walk order is always columns then rows. This just controls the sorting of the columns/rows within that "sub-walk".

Reply:
We'll probably want other walk orders in the future, but I'm fine changing the internal API when we get there.